compendium/LogService/include/cppmicroservices/logservice/LogService.hpp

```diff
@@ -69,11 +70,24 @@ namespace cppmicroservices
          *
          * @remarks This class is thread safe.
          */
-        class LogService
+        class LogService : public LoggerFactory
         {
           public:
             virtual ~LogService() = default;
 
+	    LogService() = default;
```

Comment:
this class is abstract and cannot be instantiated. There should not be a default constructor.